templates/config.yaml

```diff
@@ -591,6 +591,9 @@ templates:
   - from: templates/kyma-github-release.yaml
     render:
       - to: ../prow/jobs/kyma/kyma-github-release.yaml
+  - from: templates/kyma-gke-compass-integration.yaml
+    render:
+      - to: ../prow/jobs/kyma/kyma-gke-compass-integration.yaml
```

Comment:
I think we should add `since` declaration to not to create jobs for previous releases, as we are not sure if the would pass (in case patch release will be needed)